platform/viewer/src/lib/DICOMSR/retrieveDataFromSR.js

```diff
@@ -0,0 +1,70 @@
+import * as dcmjs from 'dcmjs';
```

Comment:
I'm not sure `retrieve` is the best word here.
And it may be a bit verbose, but expanding `SR` to `StructuredReport` may help out newcomers to DICOM-land

- `parseDicomStructuredReport()`, maybe?